modules/Book.js

```diff
@@ -0,0 +1,27 @@
+export default class Book {
+  constructor() {
+    this.books = [];
+  }
+
+  addBook(obj) {
+    this.books = JSON.parse(localStorage.getItem('data'));
+    if (this.books === null) {
+      this.books = [];
+      localStorage.setItem('data', this.books);
+      this.books.push(obj);
+      localStorage.setItem('data', JSON.stringify(this.books));
+      document.querySelector('form').reset();
+    } else {
+      this.books.push(obj);
+      localStorage.setItem('data', JSON.stringify(this.books));
+      document.querySelector('form').reset();
+    }
+  }
+
+  removeBook(id) {
+    this.books = JSON.parse(localStorage.getItem('data'));
+    this.books = this.books.filter((i) => i?.id !== id);
+    localStorage.setItem('data', JSON.stringify(this.books));
+    window.location.reload();
+  }
+}
```

Comment:
- [ ] Great effort working on the awesome book project 👍 , but currently, in your project, I noticed I could not add or remove books, anytime a user tries to add a book using the form in the `add book` section it's not adding and in the `list` section it's not displaying any added book which makes it difficult to remove a book from the list. Kindly fix this